cmd/get_test.go

```diff
@@ -367,3 +367,29 @@ func TestGetDatasetCheckedOutUsingDscache(t *testing.T) {
 		t.Errorf("unexpected (-want +got):\n%s", diff)
 	}
 }
+
+func TestGetRemoteDataset(t *testing.T) {
+	run := NewTestRunnerWithMockRemoteClient(t, "test_peer_pull_and_list", "pull_and_list")
```

Comment:
There's already a test using this testname and peername. Change to something like "test_get_remote_dataset" and "get_remote_dataset".